src/icons/BaseSvg.ts

```diff
@@ -29,7 +28,17 @@ function shouldForwardProp(prop) {
 	].includes(prop);
 }
 
-const Svg = styled.svg.withConfig({shouldForwardProp})`
+interface StyledIconBase {
+	spacingLeft?: string,
+	spacingRight?: string,
+	scale?: number,
+	vAlign?: string, // not sure about this one
+	dimmed?: boolean,
+	disablePointerEvents?: boolean,
+	flip?: number // also not sure about this one
```

Comment:
It's a string, either `'x'` or `'y'`. Would the below work? Also added the `rotate` prop that was missing.

```suggestion
	flip?: 'x' | 'y',
	rotate?: number,
```